[WIP] Cleaning of OpenPV repository

wshainin opened the pull request:
I'm creating a pull request for the cleanup branch to have a log of what I'm doing, mostly for others to look at. I'll merge it myself once I've completed the cleanup and tested it. The commits are not squashed right now, but will be when I merge.

wshainin committed:
Moved contents of pv-core into OpenPV root

And removed psycho, input, scripts, tests, and misc unused files.

wshainin committed:
Updated .gitignore with pv-core/.gitignore

wshainin committed:
Remove auxlib and move contents to Projects repo

wshainin committed:
Deleted projects directory after moving elsewhere

-renamed demo -> demos
-moved HeatMapLocalization to demos
-removed projects/AlexNet (redundant code)
-removed projects/MLPLCA (redundant code)
-moved all other projects to separate repository (PetaVision/Projects)

wshainin committed:
Merging recent changes to master into cleanup
Merge branch 'master' into cleanup

slundqui commented:
Where are you moving the demos to? There are documentation that link to
various demos, so you'll want to make sure they still correctly link to the
right place.

Sheng
On Apr 29, 2016 10:22 AM, "William Shainin" notifications@github.com
wrote:

> I'm creating a pull request for the cleanup branch to have a log of what
> I'm doing, mostly for others to look at. I'll merge it myself once I've
> completed the cleanup and tested it. The commits are not squashed right
> 
> ## now, but will be when I merge.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/PetaVision/OpenPV/pull/52
> Commit Summary
> - Moved contents of pv-core into OpenPV root
> - Updated .gitignore with pv-core/.gitignore
> - Remove auxlib and move contents to Projects repo
> - Deleted projects directory after moving elsewhere
> - Merging recent changes to master into cleanup
> 
> File Changes
> - _M_ .gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-0 (6)
> - _R_ PV-CORE_CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-1 (0)
> - _D_ auxlib/LICENSE
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-2 (204)
> - _D_ auxlib/README
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-3 (1)
> - _D_ auxlib/matchingpursuit/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-4 (1)
> - _D_ auxlib/matchingpursuit/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-5 (17)
> - _D_ auxlib/matchingpursuit/src/MatchingPursuitLayer.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-6 (295)
> - _D_ auxlib/matchingpursuit/src/MatchingPursuitLayer.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-7 (70)
> - _D_ auxlib/matchingpursuit/src/MatchingPursuitRegisterKeywords.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-8 (16)
> - _D_ auxlib/matchingpursuit/src/MatchingPursuitRegisterKeywords.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-9 (10)
> - _D_ auxlib/matchingpursuit/src/MatchingPursuitResidual.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-10 (64)
> - _D_ auxlib/matchingpursuit/src/MatchingPursuitResidual.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-11 (39)
> - _D_ auxlib/matchingpursuit/tests/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-12 (5)
> - _D_ auxlib/matchingpursuit/tests/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-13 (16)
> - _D_ auxlib/matchingpursuit/tests/MatchingPursuitTest/.project
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-14 (27)
> - _D_ auxlib/matchingpursuit/tests/MatchingPursuitTest/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-15 (2)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/input/MatchingPursuitTest.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-16 (155)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/input/testimage.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-17 (0)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/linux-cmake.cproject
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-18 (42)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/linux-eclipse.cproject
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-19 (141)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/mac-cmake.cproject
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-20 (73)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/src/MatchingPursuitProbe.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-21 (80)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/src/MatchingPursuitProbe.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-22 (38)
> - _D_
>   auxlib/matchingpursuit/tests/MatchingPursuitTest/src/MatchingPursuitTest.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-23 (19)
> - _D_ auxlib/obsolete/pvtext/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-24 (1)
> - _D_ auxlib/obsolete/pvtext/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-25 (177)
> - _D_ auxlib/obsolete/pvtext/src/PVtextGroupHandler.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-26 (47)
> - _D_ auxlib/obsolete/pvtext/src/PVtextGroupHandler.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-27 (30)
> - _D_ auxlib/obsolete/pvtext/src/TextStream.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-28 (557)
> - _D_ auxlib/obsolete/pvtext/src/TextStream.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-29 (91)
> - _D_ auxlib/obsolete/pvtext/src/TextStreamProbe.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-30 (201)
> - _D_ auxlib/obsolete/pvtext/src/TextStreamProbe.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-31 (42)
> - _D_ auxlib/pvsound/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-32 (1)
> - _D_ auxlib/pvsound/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-33 (26)
> - _D_ auxlib/pvsound/Modules/FindLIBSNDFILE.cmake
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-34 (29)
> - _D_ auxlib/pvsound/Modules/FindPackageHandleStandardArgs.cmake
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-35 (58)
> - _D_ auxlib/pvsound/src/NewCochlear.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-36 (397)
> - _D_ auxlib/pvsound/src/NewCochlear.h
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-37 (106)
> - _D_ auxlib/pvsound/src/PVsoundRegisterKeywords.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-38 (15)
> - _D_ auxlib/pvsound/src/PVsoundRegisterKeywords.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-39 (12)
> - _D_ auxlib/pvsound/src/SoundStream.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-40 (162)
> - _D_ auxlib/pvsound/src/SoundStream.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-41 (65)
> - _R_ cMakeHeader.template
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-42 (0)
> - _D_ demo/LICENSE
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-43 (204)
> - _R_ demos/AlexNet/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-44 (0)
> - _R_ demos/AlexNet/input/cifarTest.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-45 (0)
> - _R_ demos/AlexNet/input/cifarTrain.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-46 (0)
> - _R_ demos/AlexNet/input/cifarTrain.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-47 (0)
> - _R_ demos/AlexNet/scripts/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-48 (0)
> - _R_ demos/AlexNet/scripts/makeCifarPvp.py
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-49 (0)
> - _R_ demos/AlexNet/scripts/plotCost.py
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-50 (0)
> - _R_ demos/HeatMapLocalization/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-51 (0)
> - _R_ demos/HeatMapLocalization/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-52 (0)
> - _R_ demos/HeatMapLocalization/src/ConvertFromTable.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-53 (0)
> - _R_ demos/HeatMapLocalization/src/ConvertFromTable.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-54 (0)
> - _R_ demos/HeatMapLocalization/src/HarnessCustomGroupHandler.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-55 (0)
> - _R_ demos/HeatMapLocalization/src/HarnessCustomGroupHandler.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-56 (0)
> - _R_ demos/HeatMapLocalization/src/LocalizationProbe.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-57 (0)
> - _R_ demos/HeatMapLocalization/src/LocalizationProbe.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-58 (0)
> - _R_ demos/HeatMapLocalization/src/MaskFromMemoryBuffer.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-59 (0)
> - _R_ demos/HeatMapLocalization/src/MaskFromMemoryBuffer.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-60 (0)
> - _R_ demos/HeatMapLocalization/src/main.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-61 (0)
> - _R_ demos/LCACifarDemo/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-62 (0)
> - _R_ demos/LCACifarDemo/README
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-63 (0)
> - _R_ demos/LCACifarDemo/input/LCA_Cifar.lua
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-64 (0)
> - _R_ demos/LCACifarDemo/input/LCA_Cifar.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-65 (0)
> - _R_ demos/LCACifarDemo/scripts/analysis.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-66 (0)
> - _R_ demos/LICENSE
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-67 (0)
> - _R_ demos/PASCAL_Classification/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-68 (0)
> - _R_ demos/PASCAL_Classification/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-69 (0)
> - _R_ demos/PASCAL_Classification/README
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-70 (0)
> - _R_ demos/PASCAL_Classification/README-RUNREMOTE
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-71 (0)
> - _R_ demos/PASCAL_Classification/UpdateTrigger.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-72 (0)
> - _R_ demos/PASCAL_Classification/UpdateTrigger.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-73 (0)
> - _R_ demos/PASCAL_Classification/cMakeHeader.template
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-74 (0)
> - _R_ demos/PASCAL_Classification/classNames.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-75 (0)
> - _R_ demos/PASCAL_Classification/convertLandscapeToSquareAndPortrait.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-76 (0)
> - _R_ demos/PASCAL_Classification/createConfidenceTable.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-77 (0)
> - _R_ demos/PASCAL_Classification/dbscan.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-78 (0)
> - _R_ demos/PASCAL_Classification/demo-localside.tgz
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-79 (0)
> - _R_ demos/PASCAL_Classification/heatMapMontage.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-80 (0)
> - _R_ demos/PASCAL_Classification/labels/README
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-81 (0)
> - _R_ demos/PASCAL_Classification/labels/labelaeroplane192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-82 (0)
> - _R_ demos/PASCAL_Classification/labels/labelaeroplane256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-83 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbackground192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-84 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbackground256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-85 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbicycle192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-86 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbicycle256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-87 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbird192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-88 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbird256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-89 (0)
> - _R_ demos/PASCAL_Classification/labels/labelboat192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-90 (0)
> - _R_ demos/PASCAL_Classification/labels/labelboat256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-91 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbottle192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-92 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbottle256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-93 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbus192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-94 (0)
> - _R_ demos/PASCAL_Classification/labels/labelbus256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-95 (0)
> - _R_ demos/PASCAL_Classification/labels/labelcar192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-96 (0)
> - _R_ demos/PASCAL_Classification/labels/labelcar256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-97 (0)
> - _R_ demos/PASCAL_Classification/labels/labelcat192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-98 (0)
> - _R_ demos/PASCAL_Classification/labels/labelcat256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-99 (0)
> - _R_ demos/PASCAL_Classification/labels/labelchair192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-100 (0)
> - _R_ demos/PASCAL_Classification/labels/labelchair256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-101 (0)
> - _R_ demos/PASCAL_Classification/labels/labelcow192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-102 (0)
> - _R_ demos/PASCAL_Classification/labels/labelcow256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-103 (0)
> - _R_ demos/PASCAL_Classification/labels/labeldiningtable192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-104 (0)
> - _R_ demos/PASCAL_Classification/labels/labeldiningtable256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-105 (0)
> - _R_ demos/PASCAL_Classification/labels/labeldog192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-106 (0)
> - _R_ demos/PASCAL_Classification/labels/labeldog256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-107 (0)
> - _R_ demos/PASCAL_Classification/labels/labelhorse192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-108 (0)
> - _R_ demos/PASCAL_Classification/labels/labelhorse256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-109 (0)
> - _R_ demos/PASCAL_Classification/labels/labelmotorbike192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-110 (0)
> - _R_ demos/PASCAL_Classification/labels/labelmotorbike256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-111 (0)
> - _R_ demos/PASCAL_Classification/labels/labelperson192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-112 (0)
> - _R_ demos/PASCAL_Classification/labels/labelperson256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-113 (0)
> - _R_ demos/PASCAL_Classification/labels/labelpottedplant192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-114 (0)
> - _R_ demos/PASCAL_Classification/labels/labelpottedplant256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-115 (0)
> - _R_ demos/PASCAL_Classification/labels/labelsheep192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-116 (0)
> - _R_ demos/PASCAL_Classification/labels/labelsheep256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-117 (0)
> - _R_ demos/PASCAL_Classification/labels/labelsofa192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-118 (0)
> - _R_ demos/PASCAL_Classification/labels/labelsofa256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-119 (0)
> - _R_ demos/PASCAL_Classification/labels/labeltrain192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-120 (0)
> - _R_ demos/PASCAL_Classification/labels/labeltrain256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-121 (0)
> - _R_ demos/PASCAL_Classification/labels/labeltvmonitor192.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-122 (0)
> - _R_ demos/PASCAL_Classification/labels/labeltvmonitor256.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-123 (0)
> - _R_ demos/PASCAL_Classification/makethresholdpvp.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-124 (0)
> - _R_ demos/PASCAL_Classification/monitorInputImagesDir.bash
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-125 (0)
> - _R_
>   demos/PASCAL_Classification/paramsfiles/PASCAL_S1X16_1536_ICA_VOC2007_landscape.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-126 (0)
> - _R_
>   demos/PASCAL_Classification/paramsfiles/classificationEvolution.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-127 (0)
> - _R_ demos/PASCAL_Classification/paramsfiles/landscape16X12.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-128 (0)
> - _R_ demos/PASCAL_Classification/paramsfiles/landscape4X3.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-129 (0)
> - _R_ demos/PASCAL_Classification/paramsfiles/portrait12X16.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-130 (0)
> - _R_ demos/PASCAL_Classification/paramsfiles/portrait3X4.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-131 (0)
> - _R_ demos/PASCAL_Classification/paramsfiles/square16X16.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-132 (0)
> - _R_ demos/PASCAL_Classification/paramsfiles/square4X4.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-133 (0)
> - _R_ demos/PASCAL_Classification/printResultsToFile.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-134 (0)
> - _R_ demos/PASCAL_Classification/pvLandscapeDemo
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-135 (0)
> - _R_ demos/PASCAL_Classification/pvPortraitDemo
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-136 (0)
> - _R_ demos/PASCAL_Classification/pvSquareDemo
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-137 (0)
> - _R_ demos/PASCAL_Classification/runlandscape.bash
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-138 (0)
> - _R_ demos/PASCAL_Classification/runportrait.bash
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-139 (0)
> - _R_ demos/PASCAL_Classification/runsquare.bash
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-140 (0)
> - _R_ demos/PASCAL_Classification/src/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-141 (0)
> - _R_ demos/PASCAL_Classification/src/HeatMapProbe.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-142 (0)
> - _R_ demos/PASCAL_Classification/src/HeatMapProbe.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-143 (0)
> - _R_ demos/PASCAL_Classification/src/PASCALCustomGroupHandler.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-144 (0)
> - _R_ demos/PASCAL_Classification/src/PASCALCustomGroupHandler.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-145 (0)
> - _R_ demos/PASCAL_Classification/src/main.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-146 (0)
> - _R_ demos/PASCAL_Classification/src/parseConfigFile.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-147 (0)
> - _R_ demos/PASCAL_Classification/src/parseConfigFile.hpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-148 (0)
> - _R_ demos/PASCAL_Classification/src/template-config.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-149 (0)
> - _R_ demos/README
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-150 (0)
> - _R_ demos/Total_Energy_Demo/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-151 (0)
> - _R_ demos/Total_Energy_Demo/CMakeLists.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-152 (0)
> - _R_ demos/Total_Energy_Demo/HardThreshold.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-153 (0)
> - _R_ demos/Total_Energy_Demo/ISTA.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-154 (0)
> - _R_ demos/Total_Energy_Demo/README
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-155 (0)
> - _R_ demos/Total_Energy_Demo/SoftThreshold.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-156 (0)
> - _R_ demos/Total_Energy_Demo/Total_Energy_Demo.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-157 (0)
> - _R_ demos/Total_Energy_Demo/analysis/energy_graph.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-158 (0)
> - _R_ demos/Total_Energy_Demo/analysis/loadColumnEnergyProbe.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-159 (0)
> - _R_ demos/Total_Energy_Demo/analysis/loadHyPerCol_timescales.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-160 (0)
> - _R_ demos/Total_Energy_Demo/analysis/loadNormProbe.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-161 (0)
> - _R_ demos/Total_Energy_Demo/analysis/readFirmThreshold.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-162 (0)
> - _R_ demos/Total_Energy_Demo/analysis/readHardThreshold.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-163 (0)
> - _R_ demos/Total_Energy_Demo/analysis/readSoftThreshold.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-164 (0)
> - _R_ demos/Total_Energy_Demo/analysis/readenergydata.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-165 (0)
> - _R_ demos/Total_Energy_Demo/analysis/varyfirmness.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-166 (0)
> - _R_ demos/Total_Energy_Demo/input/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-167 (0)
> - _R_ demos/Total_Energy_Demo/input/AlmostHardThreshold.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-168 (0)
> - _R_ demos/Total_Energy_Demo/input/AlmostSoftThreshold.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-169 (0)
> - _R_ demos/Total_Energy_Demo/input/CIFAR_11633.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-170 (0)
> - _R_ demos/Total_Energy_Demo/input/CIFAR_13975.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-171 (0)
> - _R_ demos/Total_Energy_Demo/input/CIFAR_15921.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-172 (0)
> - _R_ demos/Total_Energy_Demo/input/CIFAR_17794.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-173 (0)
> - _R_ demos/Total_Energy_Demo/input/CIFAR_18609.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-174 (0)
> - _R_ demos/Total_Energy_Demo/input/CIFAR_frames.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-175 (0)
> - _R_ demos/Total_Energy_Demo/input/FirmThreshold.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-176 (0)
> - _R_ demos/Total_Energy_Demo/input/HardThreshold.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-177 (0)
> - _R_ demos/Total_Energy_Demo/input/ICA_Dictionary_README.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-178 (0)
> - _R_ demos/Total_Energy_Demo/input/ISTA.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-179 (0)
> - _R_ demos/Total_Energy_Demo/input/SoftThreshMasked.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-180 (0)
> - _R_ demos/Total_Energy_Demo/input/SoftThreshold.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-181 (0)
> - _R_ demos/Total_Energy_Demo/input/UsingANNNormalizedError.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-182 (0)
> - _R_ demos/Total_Energy_Demo/input/UsingQuotientColProbe.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-183 (0)
> - _R_ demos/Total_Energy_Demo/input/mask.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-184 (0)
> - _R_ demos/Total_Energy_Demo/input/mask.pvp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-185 (0)
> - _R_ demos/Total_Energy_Demo/src/main.cpp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-186 (0)
> - _R_ demos/binocDemo/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-187 (0)
> - _R_ demos/binocDemo/inputs/ATA_train_LCA.lua
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-188 (0)
> - _R_ demos/binocDemo/inputs/ATA_validate_LCA.lua
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-189 (0)
> - _R_ demos/binocDemo/inputs/binoc_learn_512_white.lua
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-190 (0)
> - _R_ demos/binocDemo/scripts/binoc.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-191 (0)
> - _R_ demos/webDemo/favicon.ico
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-192 (0)
> - _R_ demos/webDemo/images/petavisionLogo.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-193 (0)
> - _R_ demos/webDemo/index.html
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-194 (0)
> - _R_ demos/webDemo/monitor.php
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-195 (0)
> - _R_ demos/webDemo/stylesheets/styles.css
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-196 (0)
> - _R_ demos/webDemo/upload.php
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-197 (0)
> - _R_ docs/AWS/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-198 (0)
> - _R_ docs/AWS/README_INSTALL_AWS
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-199 (0)
> - _R_ docs/AWS/cpCluster.sh
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-200 (0)
> - _R_ docs/AWS/hosts
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-201 (0)
> - _R_ docs/AWS/nodefile
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-202 (0)
> - _R_ docs/AWS/setup.sh
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-203 (0)
> - _R_ docs/doxygen/.gitignore
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-204 (0)
> - _R_ docs/doxygen/buildDoxygen.sh
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-205 (0)
> - _R_ docs/doxygen/cfg/DoxygenLayout.xml
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-206 (0)
> - _R_ docs/doxygen/cfg/doxy.cfg
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-207 (0)
> - _R_ docs/doxygen/cfg/doxy_xml.cfg
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-208 (0)
> - _R_ docs/doxygen/src/main.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-209 (0)
> - _R_ docs/doxygen/src/obsolete/aws_pv_internal.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-210 (0)
> - _R_ docs/doxygen/src/obsolete/install_aws.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-211 (0)
> - _R_ docs/doxygen/src/obsolete/install_osx.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-212 (0)
> - _R_ docs/doxygen/src/obsolete/install_ubuntu.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-213 (0)
> - _R_ docs/imgs/accumulate.ai
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-214 (0)
> - _R_ docs/imgs/accumulate.pdf
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-215 (0)
> - _R_ docs/imgs/hotredcar.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-216 (0)
> - _R_ docs/imgs/neural-connectivity.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-217 (0)
> - _R_ docs/imgs/neural-fire.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-218 (0)
> - _R_ docs/imgs/neural-layers.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-219 (0)
> - _R_ docs/install/install_diagram_tool.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-220 (0)
> - _R_ docs/install/install_environment_features.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-221 (0)
> - _R_ docs/legacy-docs/BuildingFromCommandLine.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-222 (0)
> - _R_ docs/legacy-docs/InitWeightsClass.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-223 (0)
> - _R_ docs/legacy-docs/README_INSTALL_CMAKE.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-224 (0)
> - _R_ docs/legacy-docs/README_INSTALL_ECLIPSE_LINUX
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-225 (0)
> - _R_ docs/legacy-docs/README_INSTALL_ECLIPSE_MACOSX
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-226 (0)
> - _R_ docs/legacy-docs/README_INSTALL_OSX
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-227 (0)
> - _R_ docs/legacy-docs/README_INSTALL_UBUNTU
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-228 (0)
> - _R_ docs/legacy-docs/README_INSTALL_linux
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-229 (0)
> - _R_ docs/legacy-docs/extended.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-230 (0)
> - _R_ docs/legacy-docs/input_params.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-231 (0)
> - _R_ docs/legacy-docs/params-file-parameters.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-232 (0)
> - _R_ docs/publications/SY Lundquist, DM Paiton, BM Nowers, PF
>   Schultz, GT Kenyon (2013) - Biologically Inspired Distributed Sensor
>   Networks.pdf
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-233 (0)
> - _R_ docs/scripts/install_osx.sh
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-234 (0)
> - _R_ docs/tutorial/V1_LCA_tutorial.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-235 (0)
> - _R_ docs/tutorial/adaptiveTimeStep.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-236 (0)
> - _R_ docs/tutorial/adaptiveTimeStep.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-237 (0)
> - _R_ docs/tutorial/advanced/LeakyIntegrateAndFire_LIFGap.pdf
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-238 (0)
> - _R_ docs/tutorial/advanced/LeakyIntegrateAndFire_LIFGap.tex
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-239 (0)
> - _R_ docs/tutorial/basic/SLP_Classify_V1_LCA.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-240 (0)
> - _R_ docs/tutorial/basic/SLP_classify_V1_LCA_tutorial.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-241 (0)
> - _R_ docs/tutorial/basic/V1ToError_W.pvp
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-242 (0)
> - _R_ docs/tutorial/basic/V1_LCA.params
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-243 (0)
> - _R_ docs/tutorial/basic/V1_LCA_complicated.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-244 (0)
> - _R_ docs/tutorial/basic/V1_LCA_simple.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-245 (0)
> - _R_ docs/tutorial/basic/V1_LCA_tutorial.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-246 (0)
> - _R_ docs/tutorial/basic/params_file_format.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-247 (0)
> - _R_ docs/tutorial/basic/patchsizes.odg
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-248 (0)
> - _R_ docs/tutorial/basic/patchsizes.pdf
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-249 (0)
> - _R_ docs/tutorial/basic/patchsizes.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-250 (0)
> - _R_ docs/tutorial/developers/Subclassing_HyPerLayer.md
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-251 (0)
> - _R_ docs/tutorial/dtTau.png
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-252 (0)
> - _R_ docs/tutorial/patchsizemarginwidthreq.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-253 (0)
> - _R_ docs/tutorial/scale_factor.txt
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-254 (0)
> - _R_ mlab/HyPerLCA/LCA_analysis2.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-255 (0)
> - _R_ mlab/HyPerLCA/LCA_analysis3.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-256 (0)
> - _R_ mlab/HyPerLCA/LCA_brian.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-257 (0)
> - _R_ mlab/HyPerLCA/LCA_sheng.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-258 (0)
> - _R_ mlab/HyPerLCA/MNIST_analysis_britni_test.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-259 (0)
> - _R_ mlab/HyPerLCA/VL_CIFAR.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-260 (0)
> - _R_ mlab/HyPerLCA/analyzeNonSparsePVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-261 (0)
> - _R_ mlab/HyPerLCA/analyzeReconErrorPVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-262 (0)
> - _R_ mlab/HyPerLCA/analyzeReconPVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-263 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseEpochsPVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-264 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseEpochsPVP2.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-265 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseEpochsPVP3.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-266 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseHistPoolEpochsPVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-267 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseHistPoolEpochsPVP2.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-268 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseHistPoolEpochsPVP3.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-269 (0)
> - _R_ mlab/HyPerLCA/analyzeSparseHistPoolEpochsPVP4.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-270 (0)
> - _R_ mlab/HyPerLCA/analyzeSparsePVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-271 (0)
> - _R_ mlab/HyPerLCA/analyzeUnwhitenedReconPVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-272 (0)
> - _R_ mlab/HyPerLCA/analyzeXCorr.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-273 (0)
> - _R_ mlab/HyPerLCA/calcSparseHistPVPArray.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-274 (0)
> - _R_ mlab/HyPerLCA/calcSparseHistPVPArray2.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-275 (0)
> - _R_ mlab/HyPerLCA/calcSparseHistPVPArray3.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-276 (0)
> - _R_ mlab/HyPerLCA/calcSparseHistPVPArray4.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-277 (0)
> - _R_ mlab/HyPerLCA/calcSparsePVPArray.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-278 (0)
> - _R_ mlab/HyPerLCA/calcSparsePVPArray2.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-279 (0)
> - _R_ mlab/HyPerLCA/combineErrorVsSparse.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-280 (0)
> - _R_ mlab/HyPerLCA/combinePercentChangeVsTime.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-281 (0)
> - _R_ mlab/HyPerLCA/compareClassID.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-282 (0)
> - _R_ mlab/HyPerLCA/computeFiringStats.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-283 (0)
> - _R_ mlab/HyPerLCA/drawpvp_MNIST_weights.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-284 (0)
> - _R_ mlab/HyPerLCA/drawpvp_activity.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-285 (0)
> - _R_ mlab/HyPerLCA/drawpvp_weights.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-286 (0)
> - _R_ mlab/HyPerLCA/extractCIFAR.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-287 (0)
> - _R_ mlab/HyPerLCA/extractImagesOctave.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-288 (0)
> - _R_ mlab/HyPerLCA/freqmap_weights.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-289 (0)
> - _R_ mlab/HyPerLCA/getCheckpointList.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-290 (0)
> - _R_ mlab/HyPerLCA/get_Blur_weights.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-291 (0)
> - _R_ mlab/HyPerLCA/get_DoG_weights.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-292 (0)
> - _R_ mlab/HyPerLCA/invertdog.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-293 (0)
> - _R_ mlab/HyPerLCA/liblinearPVP.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-294 (0)
> - _R_ mlab/HyPerLCA/liblinearPVP2.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-295 (0)
> - _R_ mlab/HyPerLCA/liblinearPVP3.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-296 (0)
> - _R_ mlab/HyPerLCA/liblinearPVP4.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-297 (0)
> - _R_ mlab/HyPerLCA/liblinearPVP5.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-298 (0)
> - _R_ mlab/HyPerLCA/liblinearPVP6.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-299 (0)
> - _I_ mlab/HyPerLCA/liblinearPVP7.m
>   https://github.com/PetaVision/OpenPV/pull/52/files#diff-300 (0)
> 
> Patch Links:
> - https://github.com/PetaVision/OpenPV/pull/52.patch
> - https://github.com/PetaVision/OpenPV/pull/52.diff
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/PetaVision/OpenPV/pull/52

riskybacon committed:
Modifying CMake for more shallow directory structure

wshainin commented:
I'm going to remove a few of them, but the ones that are still important and actually demos will be in the demos directory. I'll update documentation accordingly, but we should go through them and make sure whatever we keep is still working.

riskybacon committed:
Tests are now a CMake option, moved PVSystemTests to tests

riskybacon committed:
Tests are now a CMake option, moved PVSystemTests to tests

wshainin committed:
Merging in recent changes to develop for testing

wshainin committed:
Removed much of mlab

wshainin committed:
Moved scripts out of OpenPV

wshainin committed:
Removed outdated PASCAL demo and moved CIFAR demo

wshainin committed:
Removed web demo

This should be VCed elsewhere until it is truly a demo that a third
party could reasonably figure out.

wshainin committed:
Moved Total_Energy_Demo to tutorials/Total_Energy

wshainin committed:
Making sure merge conflicts are trivial

Merge branch 'develop' into cleanup

wshainin committed:
Removed outdated docs and demos

wshainin committed:
Merge 'develop' into cleanup to fix conflicts

wshainin committed:
Removing old references from sample-subdirectories

wshainin commented:
There are still a few things that could be done to make the repository cleaner, and a few things that need to be done before a release, but the bulk of the cleanup is done. I'm merging cleanup with develop. The system tests pass on Ubuntu (Lenna) and OSX.
